crates/cxx-qt-gen/src/generator/rust/qobject.rs

```diff
@@ -116,6 +124,15 @@ fn generate_qobject_definitions(qobject_idents: &QObjectNames) -> Result<Generat
         }
     };
 
+    let base_upcast = if let Some(base) = base {
+        let base_name = type_names
+            .lookup(&format_ident!("{base}"))?
+            .rust_unqualified();
```

Comment:
think this should be qualified as it needs the module? as it needs to be `ffi::T` not just `T`

Reply:
I think this is still valid, instead of `        let base_name = type_names.lookup(&base)?.rust_unqualified();` it should be `        let base_name = type_names.lookup(&base)?.rust_qualified();` or something? As it needs to be `ffi::T` not `T` ?